api/component_data.go

```diff
@@ -53,8 +60,19 @@ type ComponentDataCompleteResponse struct {
 	Guid string `json:"guid,omitempty"`
 }
 
-func (svc *ComponentDataService) UploadFiles(name string, tags []string, paths []string) (string, error) {
-	initialRequest, err := buildComponentDataInitialRequest(name, tags, paths)
+func (svc *ComponentDataService) UploadFiles(
```

Comment:
this is a breaking change right? this functions is used by other components and we are changing signature. I'd prefer to avoid doing that and making the change backward compatible

Reply:
Made it backwards compatible now